content/core-features/presence.textile

```diff
@@ -11,4 +11,4 @@ index: 27
 
 h2(#occupancy-presence). Occupancy and presence
 
-"Occupancy":/core-features/occupancy differs from presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients connected to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using channel enumeration, or emitted as events when metrics change using an integration rule source or inband channel occupancy.
\ No newline at end of file
+"Occupancy":/core-features/occupancy differs from presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients attached to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using the channel status API, or emitted as events when metrics change using a @channel.occupancy@ rule source or inband channel occupancy.
```

Comment:
See comment in `occupancy.textile`.

Reply:
Applied the same change as made to `occupany.textile`.